src/main/java/ca/corefacility/bioinformatics/irida/ria/web/launchPipeline/LaunchAjaxController.java

```diff
@@ -26,11 +29,14 @@
 @RequestMapping("/ajax/pipeline")
 public class LaunchAjaxController {
     private final UIPipelineService pipelineService;
+    private final UIPipelineStartService startService;
```

Comment:
Rename `startService` to `pipelineStartService` throughout this file for clarity

Reply:
Updated in e9899f4c0462f472459a544ee840372ff725ac4c